WalletWasabi/Tor/Http/Extensions/HttpContentExtensions.cs

```diff
@@ -1,12 +1,13 @@
 using Newtonsoft.Json;
 using System.Net.Http;
 using System.Threading.Tasks;
+using WalletWasabi.WabiSabi.Models.Serialization;
 
 namespace WalletWasabi.Tor.Http.Extensions;
 
 public static class HttpContentExtensions
 {
-	private static readonly JsonSerializerSettings Settings = new();
+	private static readonly JsonSerializerSettings Settings = JsonSerializationOptions.Default.Settings;
```

Comment:
It is not used in a lot of places, AFAIU you requested for it here:
https://github.com/WalletWasabi/WalletWasabi/pull/13193#discussion_r1667388904

Reply:
It changes the way things were serialized and the only way to be sure is to go one by one checking that things are serialized exactly as before or create tests to verify that everything is serialized as before. The method is used for the synchronizer for example, which affects the fee rates, exchange rates, filters, coinjoins ids, etc. I would revert this change because it is hard to justify.

Reply:
reverted